app/src/main/java/com/onboarding/parkingsystemjava/activity/MainActivity.java

```diff
@@ -20,12 +19,17 @@ protected void onCreate(Bundle savedInstanceState) {
         binding = ActivityMainBinding.inflate(getLayoutInflater());
         setContentView(binding.getRoot());
 
-        presenter = new ParkingPresenter(new ParkingModel(), new ParkingView(this, binding));
+        presenter = new ParkingPresenter(new ParkingModel(), new ParkingView(this));
```

Comment:
why did you delete the binding?

Reply:
because ParkingPresenter didn't used it.

Reply:
ok. you mean ParkingView